myVK/myVK/ViewController/NewsTableViewController/Controller/NewsTableViewController.swift

```diff
@@ -92,6 +159,7 @@ extension NewsTableViewController {
         DispatchQueue.main.async {
```

Comment:
```suggestion
        DispatchQueue.main.async {
```
AF и так возвращает в main